packages/twenty-docs/src/pages/rest-api.tsx

```diff
@@ -0,0 +1,120 @@
+import Layout from "@theme/Layout";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+import React, { useEffect, useState } from "react";
+import { API } from '@stoplight/elements';
+import '@stoplight/elements/styles.min.css';
+import './graphql.css'
+import { parseJson } from "nx/src/utils/json";
+import { TbLoader2 } from "react-icons/tb";
+
+interface TokenFormPropsI {
+  onSubmit: (token: string) => void,
+  isTokenValid: boolean | undefined,
+  isLoading: boolean,
+  token: string,
+}
+
+const TokenForm = ({onSubmit, isTokenValid, token, isLoading}: TokenFormPropsI)=> {
+  const updateToken = (event: React.ChangeEvent<HTMLInputElement>) => {
+    localStorage.setItem('TryIt_securitySchemeValues', JSON.stringify({bearerAuth: event.target.value}))
+    onSubmit(event.target.value)
+  }
+
+  return !isTokenValid && (
+    <div>
+      <div className='container'>
+      <form className="form">
+        <label>
+          To load all your API endpoints, <a className='link' href='https://app.twenty.com/settings/developers/api-keys'>generate an API key</a> and paste it here:
+        </label>
+        <p>
+          <input
+            className={(token && !isLoading) ? "input invalid" : "input"}
+            type='text'
+            disabled={isLoading}
+            placeholder='123'
```

Comment:
let's put a placeholder that looks a bit more like a Bearer (maybe not as long though)